pkg/status/combinedstatus-resolution.go

```diff
@@ -663,6 +679,10 @@ func handleAggregation(scName string, scData *statusCollectorData) *v1alpha1.Nam
 			valuesTuple := make([]string, 0, len(scData.GroupBy))
 			for _, groupByNamedExp := range scData.GroupBy {
 				groupByValue := wsData.groupByEval[groupByNamedExp.Name]
+				if groupByValue == nil {
```

Comment:
Under what circumstances would `groupByValue == nil`?

BTW, this func's name should end with "ReadLocked", right?

Reply:
Was the wrong fix for a different no longer present bug, fixed.